pkg/cmd/cli/describe/describer.go

```diff
@@ -345,6 +345,15 @@ func describeCustomStrategy(s *buildapi.CustomBuildStrategy, out *tabwriter.Writ
 }
 
 func describeJenkinsPipelineStrategy(s *buildapi.JenkinsPipelineBuildStrategy, out *tabwriter.Writer) {
+	if len(s.JenkinsfilePath) != 0 {
+		formatString(out, "Jenkinsfile path", s.JenkinsfilePath)
+	}
+	if len(s.Jenkinsfile) != 0 {
+		formatString(out, "Jenkinsfile contents", s.Jenkinsfile)
```

Comment:
we may not want to print the full contents...  do we print the full Dockerfile when the buildconfig has an in-line dockerfile definition?